MIP104/MIP104.md

```diff
@@ -209,207 +209,35 @@ Advisory Council project budget:
-
-## 4: AllocatorDAO Vaults
-
-Before the launch of SubDAOs, AllocatorDAO Vaults function in a bootstrapping mode. They are controlled by AllocatorDAO Advisors, which are special Ecosystem Actors with the ability to advise the Stability Facilitators
-
-### 4.1: AllocatorDAO Vaults Stability Fee
-
-The AllocatorDAO Vaults have a Stability Fee equal to the Base Rate specified in *3.1.1A*.
-
-### 4.2: AllocatorDAO Advisors
-
-The Stability Facilitator can use the weekly governance cycle to propose to onboard AllocatorDAO Advisors and set up a new AllocatorDAO Vault, based on AllocatorDAO Advisor Proposals made on the Maker Forum. The AllocatorDAO Advisor will be tasked with publishing advice related to the use of their AllocatorDAO vault. The current active AllocatorDAO Advisers and their individual Debt Ceiling Limits are specified in *4.2.2A*, and must be updated by the Stability Facilitators when applicable.
-
-#### 4.2.1
-
-The AllocatorDAO Vaults share a total debt ceiling limit equivalent to the Dai Supply, minus all legacy collateral. Each AllocatorDAO Vault has an individual debt ceiling limit of its proportional amount of the total debt ceiling limit.
-
-#### 4.2.2A
-
-¤¤¤
-
-Current active AllocatorDAO Advisors:
-
-N/A
-
-¤¤¤
-
-## 5: Legal Recourse Assets
-
-Legal Resource Assets are assets used as collateral for the Dai Stablecoin that are enforced through legal recourse by Arranged Structures. They have unique risks that must be safeguarded against.
+Real World Assets are assets used as collateral for the Dai Stablecoin that are enforced through legal recourse by Arranged Structures. They have unique risks that must be safeguarded against.
 
 ### 5.1: Arrangers
 
-Arrangers are Ecosystem Actors that specialize in sourcing, negotiating, structuring of, and reporting on, Legal Recourse Assets and maintaining and monitoring the underlying Arranged Structures used by the Maker Protocol. They must never be able to operate or influence the arranged legal and operational structure’s asset operations in a way that can cause any significant harm or losses to Dai’s stability, after the structures have been built and assets allocated hereto. Arrangers are approved directly by MKR voters, and all LRA collateral exposure must be structured by an approved Arranger.
+Arrangers are Ecosystem Actors that specialize in sourcing, negotiating, structuring of, and reporting on, Real World Assets and maintaining and monitoring the underlying Arranged Structures used by the Maker Protocol. They must never be able to operate or influence the arranged legal and operational structure’s asset operations in a way that can cause any significant harm or losses to Dai’s stability, after the structures have been built and assets allocated hereto. Arrangers are approved directly by MKR voters, and all LRA collateral exposure must be structured by an approved Arranger.
```

Comment:
LRA collateral exposure (last sentence) should be changed to RWA collateral exposure, no?

Reply:
@CivicSage3 , I think that should be "LRW" (Low-Risk RWA)